src/components/EditableTextarea/EditableTextarea.tsx

```diff
@@ -96,7 +98,6 @@ export class EditableTextarea extends React.PureComponent<
   }
 
   componentWillReceiveProps(nextProps) {
-    if (nextProps.value === this.props.value) return
```

Comment:
Short circuiting here prevents resetting the component to its original state which is sometimes necessary. As such, I removed this guard.